proxy/envoy/config.go

```diff
@@ -162,6 +164,12 @@ func buildListeners(env proxy.Environment, sidecar proxy.Sidecar) (Listeners, Cl
 		insertMixerFilter(listeners, instances, sidecar)
 	}
 
+	// Append management listeners after injecting mixer filter to normal listeners
+	// because management port listeners cannot use mixer or auth (see comment
+	// under buildMgmtPortListeners)
+	listeners = append(listeners, mgmtListeners...)
```

Comment:
The previous iteration would de-duplicate management and inbound listeners. Can we at least skip management listeners that are also inbound listeners? This used to cause segfaults in Envoy before LDS.